ui/lib/utils/table-columns.tsx

```diff
@@ -125,21 +130,37 @@ export function useFieldsTimeValueColumn(
   }
 }
 
-export function useFieldsDescriptionColumn(translationPrefix: string): IColumn {
-  const { t } = useTranslation()
+function fieldsDescriptionColumn(transKeyPrefix: string): IColumn {
   return {
-    name: t('component.commonColumn.desc'),
+    name: (<TransText transKey={`${DEF_TRANSKEY_PREFIX}desc`} />) as any,
     key: 'description',
     minWidth: 150,
     maxWidth: 300,
     isResizable: true,
     columnActionsMode: ColumnActionsMode.disabled,
     onRender: (rec) => {
-      // Fallback to no language
-      return t(`${translationPrefix}${rec.key}_tooltip`, {
-        defaultValue: '',
-        fallbackLng: '_',
```

Comment:
`fallbackLng` must be preserved, otherwise you will see description in other language (due to fallback) even if we don't want a description (not setting tooltip in current language)

Reply:
soga!